README.md

```diff
@@ -129,6 +129,36 @@ val house2: House = house.copy {
 }
 ```
 
+## kopykat comparison
+
+Kopy uses the K2 compiler plugin meanwhile `kopykat` uses KSP to generate code, as consequence there are
+different advantages and disadvantages.
+
+### Advantages
+
+- Kopy generation is faster than `kopykat`.
+- Kopy does not need to run any Gradle task to get feedback on the IDE:
+    - The autocomplete shows the `copy` and `invoke` functions instantaneously after annotating a
+      data class with `@Kopy`.
+    - Removing the `@Kopy` will remove instantaneously the `copy` and `invoke` functions without
+      running a Gradle task or deleting manually the `build` directory of a project.
+    - As it is not necessary to assemble/build the project, the feedback loop is faster.
+- Build time should be better too (not tested).
+- Kopy only adds 5 or 6 functions/properties to each data class, meanwhile `kopykat` needs to
+  generate builders and the functions/properties match the number of properties in the data class.
+  In the future, the number of properties will be reduced to 1 or 2.
+
+### Disadvantages
+
+Similar to Kotlin Power Assert, this plugin works on the call site, so it modifies the body of the
```

Comment:
Please link "Kotlin Power Assert" to https://github.com/bnorm/kotlin-power-assert.